src/styles/colors.css

```diff
@@ -35,9 +35,18 @@
   --bkgrnd-lighter: #E9EFF1;
   --bkgrnd-lightest: #F8F9FA;
 
+  --tooltip-title-bg: #EAEAEE;
+
   --chrome: #989897;
 
   --text-medium-contrast: #727375;
   --text-high-contrast: #281946;
+
+  --bg-low: #FF8B7C;
+  --bg-target: #76D3A6;
+  --bg-high: #BB9AE7;
+
+  --smbg-dayline: var(--data-purple--medium);
```

Comment:
Why do you need another variable name for this? We just use `--data-purple--medium` directly in other places in the code (like the cbg side of trends view).